### PR TITLE
added applyUnique method to Compiler

### PR DESCRIPTION
**Pull request by hulkish**
**What kind of change does this PR introduce?**
Patch or Minor

feature

**Did you add tests for your changes?**
Yes

**If relevant, link to documentation update:**
TBD

**Summary**
https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/72

**Does this PR introduce a breaking change?**
No.

**Other information**

### CODE REVIEW

File: test/Compiler.test.js [hunk 1]
```diff
@@ -218,6 +218,18 @@ describe("Compiler", () => {
 				done();
 			});
 		});
+		describe("qpplyUnique", () => {
```
Comment (sendilkumarn) [inline test/Compiler.test.js]: typo

**Comment by mikesherov** [comment]
IMO, this PR seems a bit premature given that we're still discussing the best way to handle accidental plugin duplication.

**Comment by hulkish** [comment]
@mikesherov Really just wanted to make the pr as a way to discussing it a bit - and determine if there's a  better way.

**Comment by sokra** [comment]
I guess the best solution is a bigger tapable refactoring.

**Pull request closed without merging**